### ISSUE TITLE
Failed to get correct nlohmann_json dependencies

### ISSUE THREAD

**Issue by gsohler** [jupyter-xeus/xeus-zmq#77]
I cant appear to satisfy the correct  nlohmann_json dependencies.
the cmake appears to require  3.11.2 and 3.11.3 a the same time:

![image](https://github.com/user-attachments/assets/584e570b-85f3-4b66-a93f-d1fb49175d53)
![image](https://github.com/user-attachments/assets/34ad3c4c-59f0-4421-9df8-4b50716154d9)

What is correct here and how can i fix that ?

**Comment by kaelingre** [comment]
You have to recompile xeus with the newest nlomann_json version 3.11.3 (i.e. cleanbuild it again). The error is coming from the xeus config file which requires that anything that is built against it is using with the same version. (You have built xeus with nlomann_json 3.11.2 and are now trying to built xeus-zmq with a newer version 3.11.3)

**Comment by anutosh491** [comment]
Hi, I shall look into this soon and get back if there is any issue. Thanks

**Comment by gsohler** [comment]
yes that worked, i just did not know where the wrong  infomation was hidden.
thank you!
